.gitlab-ci.yml

```diff
@@ -62,3 +62,32 @@ build_wheel:
     # create the wheel and upload to pypi
     - sh scripts/create_assembl_wheel.sh
     - twine upload -u bluenove --skip-existing dist/*
+
+deploy_to_dev:
+  stage: deploy_to_dev
+  only:
+    - tmp_master_for_ci_cd
+    - master
+  script:
+    # empty script for now
+    - sh scripts/echo.sh
+
+deploy_to_uat:
+  stage: deploy_to_uat
+  only:
+    - tmp_master_for_ci_cd
+    - master
+  script:
+    # empty script for now
+    - sh scripts/echo.sh
+
+deploy_to_prod:
+  stage: depoy_to_prod
+  environment: PROD
+  only:
+    - deploy_to_uat
```

Comment:
deploy_to_uat is a tag, a branche...???

Reply:
:+1: it should be `tmp_master_for_ci_cd` here

Reply:
🤦‍♂️ I think I'm really tired today, sorry for the mess, will fix this